packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx

```diff
@@ -734,7 +735,10 @@ const PublishAction: DocumentActionComponent = ({
           // Update, if needed, the parent relation with the newly published document
           // check if in history we have the parent relation otherwise use the rootDocument
           if (fieldToConnect && documentHistory && parentDocumentMetaToUpdate.documentId) {
-            const parentDataToUpdate = parentDocumentData.getInitialFormValues();
+            const parentDataToUpdate =
+              parentDocumentMetaToUpdate.collectionType === SINGLE_TYPES
+                ? getInitialFormValues()
+                : parentDocumentData.getInitialFormValues();
```

Comment:
I wonder why `parentDocumentData.getInitialFormValues` doesn't work for single types? Asking because ideally I'd prefer to avoid needing useDoc since it's not flexible

Reply:
I receive an empty object with single types

Reply:
and then I realized that I can't have a single type document in the modal but only in the Edit view page so I can use useDoc